### PR TITLE
feat: birthday upgrades

### PR DESCRIPTION
**Pull request by StrangeGirlMurph**

### CODE REVIEW

File: src/commands/BirthdayCommand.ts [hunk 1]
```diff
@@ -66,7 +66,10 @@ class BirthdayCommand extends Command {
 					.addBooleanOption((opt) => opt.setName("delete").setDescription("Delete your birthday entry."))
 			)
 			.addSubcommand((option) =>
-				option.setName("my").setDescription("Show what date is stored for your birthday.")
+				option
+					.setName("get")
+					.setDescription("Show what date is stored for a members birthday.")
+					.addUserOption((opt) => opt.setName("user").setDescription("The user to get the birthday of."))
```
Comment (WilsontheWolf) [inline src/commands/BirthdayCommand.ts:72]: I'd make this optional and just use the current user if not specified.

**Review by WilsontheWolf — CHANGES_REQUESTED** [review]

File: src/commands/BirthdayCommand.ts (hunk 1 above)
Comment (StrangeGirlMurph) [reply src/commands/BirthdayCommand.ts:72]: yeah good idea

**Review by WilsontheWolf — APPROVED** [review]

**Review by Chicken — APPROVED** [review]

**Comment by Chicken** [comment]
nice

**Comment by WilsontheWolf** [comment]
nice

**Pull request merged**